apps/site/pages/components/molecules/toast.mdx

```diff
@@ -200,7 +200,34 @@ import '@faststore/ui/src/components/molecules/Toast/styles.scss'
 
 ### Toast
 
-<PropsSection name="Toast" />
+export const propsToast = [
+  {
+    name: 'title',
+    type: 'string',
+    description: "Toast's title.",
+  },
+  {
+    name: 'message',
+    type: 'string',
+    description: 'Message shown in Toast.',
+    required: true,
+  },
+  {
+    name: 'status',
+    type: "'ERROR' | 'WARNING' | 'INFO'",
```

Comment:
does it need to be all caps?

Reply:
I think so, it's how we defined before.

Reply:
yes, that's the way it's defined in the `UIProvided`.